openreview/venue/process/assignment_pre_process.js

```diff
@@ -31,7 +31,7 @@ async function process(client, edge, invitation) {
     // Filter invite assignment edges to exclude edges that are accepted
     const filteredInviteAssignmentEdges = inviteAssignmentEdges.filter(e => !filteredLabels.includes(e?.label ?? ''))
 
-    if (quota && filteredInviteAssignmentEdges.length + filteredAssignmentEdges.length >= quota) {
+    if (quota && filteredInviteAssignmentEdges.length + filteredAssignmentEdges.length >= quota + 1) {
```

Comment:
if you relax this validation, all the papers can have quota + 1 assignments, is this what we want?

Reply:
There are two options to solve this:

1. Reject the assignment for the invited reviewer that accepted the invitation. Send an email saying the paper already has enough reviewers and their review is not needed anymore.
2. Bypass the validation here by checking that if that assignment is coming from an invite assignment then let it post the edge. You can check that is the edge that we are trying to post already exists as an invite assignment (same head and tail) then we let the edge to be posted.